core/src/Platforms/net45/NetDesktopCryptographyManager.cs

```diff
@@ -36,16 +36,7 @@ internal class NetDesktopCryptographyManager : ICryptographyManager
     {
         public string CreateBase64UrlEncodedSha256Hash(string input)
         {
-            if (string.IsNullOrEmpty(input))
-            {
-                return null;
-            }
-
-            using (SHA256Cng sha = new SHA256Cng())
-            {
-                UTF8Encoding encoding = new UTF8Encoding();
-                return Base64UrlHelpers.Encode(sha.ComputeHash(encoding.GetBytes(input)));
-            }
+            return string.IsNullOrEmpty(input) ? null : Base64UrlHelpers.Encode(CreateSha256HashBytes(input));
```

Comment:
>IsNullOrEmpty [](start = 26, length = 13)

In the iOS and Android versions you have `.IsNullOrWhiteSpace` here.

Reply:
good catch.  fixed